api/src/main/java/org/infinispan/api/exception/InfinispanConfigurationException.java

```diff
@@ -0,0 +1,10 @@
+package org.infinispan.api.exception;
+
+/**
+ * Exception raised when a configuration error is found
+ *
+ * @since 10.0
+ * @author Katia Aresti, karesti@redhat.com
+ */
+public class InfinispanConfigurationException extends RuntimeException {
```

Comment:
This should extend InfinispanException. Better to have a single root

Reply:
ok